maestro_worker_python/convert_files.py

```diff
@@ -92,47 +92,49 @@ def convert_files_manager(*convert_files: FileToConvert) -> None | str | list[st
             obj.close()
 
 
-def _convert_to_wav(input_file_path, output_file_path, max_duration, sample_rate=44100):
+def _convert_to_wav(input_file_path: str, output_file_path: str, max_duration: str, sample_rate: int | None = 44100):
+    command_list = [
+        "ffmpeg",
+        "-y",
+        "-hide_banner",
+        "-loglevel",
+        "error",
+        "-t",
+        str(max_duration),
+        "-i",
+        str(input_file_path),
+        str(output_file_path),
+    ]
+    if sample_rate is not None:
+        command_list[9:9] = ["-ar", str(sample_rate)]
     _run_subprocess(
-        [
-            "ffmpeg",
-            "-y",
-            "-hide_banner",
-            "-loglevel",
-            "error",
-            "-t",
-            str(max_duration),
-            "-i",
-            str(input_file_path),
-            "-ar",
-            str(sample_rate),
-            str(output_file_path),
-        ]
+        command_list
     )
 
 
-def _convert_to_m4a(input_file_path, output_file_path, max_duration, sample_rate=44100):
+def _convert_to_m4a(input_file_path: str, output_file_path: str, max_duration: str, sample_rate: int | None = 44100):
+    command_list = [
```

Comment:
Thinking that we might add/remove parameters in the future, an alternative that doesn't rely on the index staying the same, or needing to remember updating it, would be:

```python
    command = [
        "ffmpeg", "-y", "-hide_banner", "-loglevel", "error",
        "-t", str(max_duration),
        "-i", str(input_file_path)
    ]
    
    if sample_rate is not None:
        command.extend(["-ar", str(sample_rate)])
    
    command.append(str(output_file_path))
    ...
```

Reply:
I selected more lines than intended, but you get the gist